package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "linkfree",
-  "version": "0.12.0",
+  "version": "0.16.3",
```

Comment:
Did you modify this directly or was it done automatically?

Reply:
I did not touch anything other than adding my json file in data dir

Reply:
@Panquesito7 it seems to do this with others too.